File: pset1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    //declare variables
    int quarter;
    int dime;
    int nickel;
    int penny;
    int changeoutput;

    float getchange;

    //prompt for change float
    do
    {
        float getchange = get_float("Amount of change: ");
    }
    
    //create loop until positive number is created
    while (getchange > 0);

    //convert to cents
    int cents = round(getchange * 100)

    //loop for quarters
    for (quarter = 1; cents >= 25; quarter++)
    {
        cents = cents - 25;
        changeoutput = change output + 1;
    }

    //loop for dimes
    for (dime = 1; cents >= 10; dime++)
    {
        cents = cents - 10;
        changeoutput = changeoutput + 1;
    }

    //loop for nickels
    for (nickel = 1; cents >= 5; nickel++)
    {
        cents = cents - 5;
        changeoutput = changeoutput + 1;
    }

    //loop for pennies
    for (penny = 1; cents >= 1; penny++)
    {
        cents = cents - 1;
        changeoutput = changeoutput + 1;
    }

    //print output
    printf("%i\n", changeoutput);
    
}

File: pset6/cash/cash.py
from cs50 import get_float

#create true loop to continue prompting user until we like their input
while True:
    #ask for amount of change
    change_owed = get_float("Change owed: ")
    if change_owed > 0:
        break

#multiply change by 100 in order to make it an int instead of float to avoid problems
cents = round(change_owed * 100)

#declare new int for coin counter
coin_counter = 0

#loop through for quarters
while True:
    if cents >= 25:
        cents = cents - 25
        coin_counter += 1
    else:
        break

#loop through for dimes
while True:
    if cents >= 10:
        cents = cents - 10
        coin_counter += 1
    else:
        break

#loop through for nicks
while True:
    if cents >= 5:
        cents = cents - 5
        coin_counter += 1
    else:
        break

#same shit for pennies
while True:
    if cents >= 1:
        cents = cents - 1
        coin_counter += 1
    else:
        break
    
#print coin counter
print(coin_counter)


